bank_classes/account_class.rb

```diff
@@ -15,11 +16,11 @@ class Account
   attr_reader :balance
 
 # - A new account should be created with an ID and an initial balance
-    def initialize
+    def initialize(open_deposit_min = 0)
       array_of_accounts = CSV.read("./support/accounts.csv")
       @account_num = array_of_accounts[@@number_of_accounts][0]
       initial_deposit = array_of_accounts[@@number_of_accounts][1]
-      @balance = open_deposit_check(initial_deposit.to_i)
+      @balance = open_deposit_check(initial_deposit.to_i, open_deposit_min)
```

Comment:
Do you want the "user" of your accounts to be able to specify their own minimum deposit amount?